docs/clusterdefinition.md

```diff
@@ -173,7 +173,8 @@ Below is a list of kubelet options that acs-engine will configure by default:
 |"--cluster-domain"|"cluster.local"|
 |"--pod-infra-container-image"|"pause-amd64:*version*"|
 |"--max-pods"|"30", or "100" if using kubenet --network-plugin (i.e., `"networkPlugin": "kubenet"`)|
-|"--eviction-hard"|"memory.available<100Mi,nodefs.available<10%,nodefs.inodesFree<5%"|

```

Comment:
It seems `100Mi` is used more often for memory, `memory.available<100Mi` is ok to me